test/unit/services/appealService.test.js

```diff
@@ -0,0 +1,189 @@
+const { getAppeal, changeEmailAddress, stopReceivingEmails } = require('app/services/appealService');
+const { expect, sinon } = require('test/chai-sinon');
+const nock = require('nock');
+const { appealsAPI } = require('app/config');
+const HttpStatus = require('http-status-codes');
+const mockAppeal = require('test/mock/data').md002.appeal;
```

Comment:
Can you require the actual file please like so:  `require('test/mock/data/appealReceived').appeal` otherwise you would need to know what `md002` refers to.